templates/index.ts.hbs

```diff
@@ -7,10 +7,14 @@ export {
 {{/each}}
 }
 
+{{!-- TODO: consider resolving the helpers as one --}}
+{{!-- NOTE: helpers is deprecated since it uses shopperCustomer --}}
```

Comment:
Suggestion:
1. Create `src/static`, like the iso sdk has. When rendering the templates, just copy the static files to the output directory.
2. Create a `src/static/helpers` directory.
3. `src/static/helpers/slas.ts` should have the SLAS helper.
4. `src/static/helpers/shopperCustomers.ts` or whatever should have the old helper. (It never changes, it doesn't need to be generated.)
4.b. Clean up the [metadata](https://github.com/SalesforceCommerceCloud/commerce-sdk/blob/master/apis/.metadata.json) that we don't need any more. (Side note, we can probably also clean up the `enforceCcdcId` stuff now, too.)
5. `src/static/helpers/index.ts` should `export * from "./whatever.ts"`
6. `src/index.ts` could either `export * as helpers from "./helpers"` or `export * from "./helpers"`. Not sure which is better.

Also, wrap the deprecated helper with [`util.deprecate`](https://nodejs.org/docs/latest-v12.x/api/util.html#util_util_deprecate_fn_msg_code).

Reply:
This cleans up what I was saying below that these things shouldn't be split between and static and template.  Well articulated well.